src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -64,6 +68,10 @@ nginx_sites:
        proxy_next_upstream_tries: 1
        proxy_read_timeout: 900s
        proxy_cache: app_cache
+       limit_reqs:
+         - zone_name: "multimedia"
```

Comment:
From #server-fires it sounds like you also added something around limiting video files specifically? Or maybe that was just a suggestion of what we could do?

Reply:
Yes, we could split this out more. Another option would be to limit the bandwidth consumption here as well as just the requests per second:

https://nginx.org/en/docs/http/ngx_http_core_module.html#limit_rate